bridges/snowbridge/primitives/test-helper/Cargo.toml

```
@@ -0,0 +1,64 @@
+[package]
+name = "snowbridge-test-helper-primitives"
```

After some thought, I think we should change the name to be less verbose.

```suggestion
name = "snowbridge-test-utils"
```